spec/spec_helper.rb

```diff
@@ -17,6 +22,19 @@
   # rspec-expectations config goes here. You can use an alternate
   # assertion/expectation library such as wrong or the stdlib/minitest
   # assertions if you prefer.
+
+  path =  File.join(
+            File.dirname(File.dirname(File.absolute_path(__FILE__))),
+            '/app/assets/json/api_response_a.json')
+  file = File.read(path)
+
+  config.before(:each) do
+    stub_request(:get, /rummager.dev.gov.uk/).
```

Comment:
I would prefer to avoid a global stub, we only need this in one test file, let's just load it there. That way when we write another test where we want a different stubbed result it is easier to do.